af_rest_appointment/models/af_appointment.py

```diff
@@ -146,59 +178,84 @@ def get_occasions(self, from_date, to_date, appointment_channel, appointment_typ
                     'name': occ_id,
                     'stop': stop_datetime,
                     'start': start_datetime,
-                    'duration': (stop_datetime - start_datetime).seconds//60 # get length in minutes
+                    # get length in minutes
+                    'duration': (stop_datetime - start_datetime).seconds // 60
                     # TODO: implement these
                     # '': occasion.get('appointment_channel'),
                     # '': occasion.get('occasion_status_id'),
                 }
                 self.env['calendar.occasion'].create(vals)
 
     # /appointments
-    def get_appointments(self, from_date, to_date, user = '', pnr = '', appointment_types = [], status_list = []):
-        client_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_id')
-        client_secret = self.env['ir.config_parameter'].sudo().get_param('af_rest.client_secret')
-        af_environment = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_environment')
-        af_port = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_port')
-        af_url = self.env['ir.config_parameter'].sudo().get_param('af_rest.ipf_url')
-        af_system_id = self.env['ir.config_parameter'].sudo().get_param('af_rest.af_system_id')
-
-        if not (af_url or af_port or client_id or client_secret or af_environment or af_system_id):
+    def get_appointments(
+            self,
+            from_date,
+            to_date,
+            user='',
+            pnr='',
+            appointment_types=[],
+            status_list=[]):
+        client_id = self.env['ir.config_parameter'].sudo(
```

Comment:
This is almost unreadable. could you loook into any other way of getting these lines to go through Travis?